falcon/ms_io/mzxml_io.py

```diff
@@ -50,16 +52,23 @@ def _parse_spectrum(spectrum_dict: Dict) -> sus.MsmsSpectrum:
     MsmsSpectrum
         The parsed cluster.
     """
-    spectrum_id = spectrum_dict['id']
-    mz_array = spectrum_dict['m/z array']
-    intensity_array = spectrum_dict['intensity array']
-    retention_time = spectrum_dict.get('retentionTime', -1)
+    spectrum_id = spectrum_dict["id"]
+    mz_array = spectrum_dict["m/z array"]
+    intensity_array = spectrum_dict["intensity array"]
+    retention_time = spectrum_dict.get("retentionTime", -1)
 
-    precursor_mz = spectrum_dict['precursorMz'][0]['precursorMz']
-    if 'precursorCharge' in spectrum_dict['precursorMz'][0]:
-        precursor_charge = spectrum_dict['precursorMz'][0]['precursorCharge']
+    precursor_mz = spectrum_dict["precursorMz"][0]["precursorMz"]
+    if "precursorCharge" in spectrum_dict["precursorMz"][0]:
+        precursor_charge = spectrum_dict["precursorMz"][0]["precursorCharge"]
     else:
-        raise ValueError('Unknown precursor charge')
+        precursor_charge = -1
```

Comment:
**todo:** Same for default precursor charge.

Reply:
Done